cli/info.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// infoDespscription is used to describe info command in detail and auto generate command doc.
+var infoDescription = "Display the information of pouch, " +
+	"including Containers, Images, Storage Driver, Execution Driver, Logging Driver, Kernel Version," +
+	"Operating System, CPUs, Total Memory, Name, ID."
+
+// InfoCommand use to implement 'info' command.
+type InfoCommand struct {
+	baseCommand
+}
+
+// Init initialize info command.
+func (v *InfoCommand) Init(c *Cli) {
+	v.cli = c
+	v.cmd = &cobra.Command{
+		Use:   "info [OPTIONS]",
+		Short: "Display system-wide information",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return v.runInfo()
+		},
+		Example: infoExample(),
+	}
+	v.addFlags()
```

Comment:
No extra flags, so we may not need addFlags function

Reply:
Will we add flag --format here? It goes like 'flags.StringVarP(&opts.format, "format", "f", "", "Format the output using the given Go template")'.

Reply:
I check docker info command, it also has no options, so i think we no need to add --format flag.